setup.py

```diff
@@ -16,7 +16,7 @@ def read(filename):
 
 setup(
     name="amsterdam-schema-tools",
-    version="0.13.2",
+    version="0.13.3",
```

Comment:
Ik zit aan het denken of dit is groot genoeg om minor versie te bumpen naar `0.14.1`? @jjmurre wat denk jij?

Reply:
Ja, lijkt mij zeker groot genoeg!